cli/accounts.go

```diff
@@ -8,14 +8,14 @@ import (
 )
 
 func init() {
-	accountsCmd.Flags().StringVar(&identityProvider, "identity-provider", defaultIdentityProvider, "The identity provider to use. Refer to `keyconjurer identity-providers` for more info.")
+	accountsCmd.Flags().StringVar(&identityProvider, "identity-provider", defaultIdentityProvider, "The identity provider to use. Refer to `"+appname+" identity-providers` for more info.")
```

Comment:
Not a big deal but I am not sure that we really need to use `appname` in strings like here. For me, it only makes it look more complicated due to additional string concatenations. Also, one can define an alias like `kc` so that effectively `appname` would be different for them.

Reply:
I noticed `BINARY_NAME` in the `example.env` file, but I don't see it integrated into the cli.  I was suspecting we could override the const with this value for users who decide to name the executable something else other than 'keyconjurer'.  Having it defined in one place makes it easier to change globally.
https://github.com/RiotGames/key-conjurer/blob/master/example.env#L16